src/content/structured/styles/icons.mdx

```diff
@@ -33,3 +33,8 @@ Consider alternatives such as dropdowns, merged buttons and splitting a layout a
 Provide a text label when you use icons. Even if the text isn't visible on the screen, assistive technology needs to describe the icon.
 
 Use tooltips on icons or icon buttons so that the meaning of an icon is obvious.
+
+## Slotted SVGs
+
```

Comment:
Does this need a tiny line to explain why the links have been put here? Something like
```suggestion
To use these Material Design Icons in a React project, please see the following guides:
```
I don't think that's worded the best, but hopefully it gives the gist of what I mean [😊](https://emojipedia.org/smiling-face-with-smiling-eyes/)